packages/react-devtools-core/src/backend.js

```diff
@@ -138,6 +138,7 @@ function setupBackend(wall, resolveRNStyle) {
   var bridge = new Bridge(wall);
   var agent = new Agent(window, {
     rnStyle: !!resolveRNStyle,
+    rnStyleMeasure: !!resolveRNStyle,
```

Comment:
Can you explain why you can’t call `measureStyle` directly in the frontend where you display it instead of going through the bridge? (I’m not sure I understand this part of React DevTools so bear with me.) It would be cool if it “just worked” with old RN versions too.